tests/interpolation/built_in_resolvers/test_oc_select.py

```diff
@@ -0,0 +1,93 @@
+from omegaconf import OmegaConf
```

Comment:
(not related to this file)

Shouldn't `test_dict.py` be renamed into `test_oc_dict.py` for consistency?

Reply:
yes.